packages/manager/src/features/Backups/BackupsCTA.tsx

```diff
@@ -50,6 +50,7 @@ export const BackupsCTA = () => {
       </Typography>
       <Box component="span" display="flex">
         <StyledLinkButton
+          aria-label="Dismiss Enable Linode Backups notice"
```

Comment:
I did not check to see if this phrasing is consistent with the labels on our other dismissal buttons

Reply:
I have not checked on this yet either, but it made me think of an [article](https://uxdesign.cc/i-counted-8-mistakes-in-this-15-word-dialog-design-21e2f4bb7a73) I was reading yesterday:

> Writing two verbs, side-by-side, is confusing. In addition, the verb “cancel” is destructive and the verb “add” is constructive, exacerbating the confusion.

And if we do phrase things this way in labels, it's probably best to avoid that going forward and fix it where we can. 

```suggestion
          aria-label="Dismiss notice enabling Linode backups"
```

Reply:
@mjac0bs Wow, that didn't occur to me but now that you've pointed it out it's obvious how unclear the existing phrasing is. Thanks for sharing!